index.html

```diff
@@ -1,3 +1,4 @@
+/*  my html code start from here */
```

Comment:
- [ ] Kindly remove this piece of text from your work as it is currently showing on the UI and it is needed. 👍

Reply:
@brainconnect93  Noted with thanks, for the next project, I am going to improve on that.

Reply:
@brainconnect93 Noted all the feedback. Thank you